### PR TITLE
added shield

### PR DESCRIPTION
**Pull request by dominikwilkowski**
Signed-off-by: Dominik Wilkowski <Hi@Dominik-Wilkowski.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,8 +94,8 @@ The design system for Westpac GEL
 ## TODO
 
 - [x] jest tests setup for each component
-- [ ] cypress test setup for each component
-- [ ] add circleCI
-- [ ] connect brand/wbc packages to core component dynamic import
+- [/] cypress test setup for each component
```
Comment (jossmac) [inline README.md]: hey this looks dodgy!

File: README.md [hunk 2]
```diff
@@ -94,8 +94,8 @@ The design system for Westpac GEL
 ## TODO
 
 - [x] jest tests setup for each component
-- [ ] cypress test setup for each component
```
Comment (jossmac) [inline README.md]: oh what!

File: README.md [hunk 3]
```diff
@@ -94,8 +94,8 @@ The design system for Westpac GEL
...
-- [ ] add circleCI
-- [ ] connect brand/wbc packages to core component dynamic import
+- [/] cypress test setup for each component
+- [x] add circleCI
 - [ ] write more docs around each decision me made
+- [ ] connect brand/wbc packages to core component dynamic import
```
Comment (jossmac) [inline README.md]:
```suggestion
- [x] connect brand/wbc packages to core component dynamic import
```

**Comment by dominikwilkowski** [comment]
@jossmac done the deed!

**Pull request merged**